cmd/skaffold/app/cmd/render.go

```diff
@@ -41,6 +45,8 @@ func NewCmdRender() *cobra.Command {
 		WithCommonFlags().
 		WithFlags(func(f *pflag.FlagSet) {
 			f.BoolVar(&showBuild, "loud", false, "Show the build logs and output")
+			f.VarP(&fromBuildOutputFile, "build-artifacts", "a", "Don't build, but use file created with `skaffold build --file-output`")
```

Comment:
this feels weird to use: I feel like with this flag I'd be providing the list of artifacts on the command line, not a file. can we change this to something like `--build-file`?

Reply:
also, for the description I'd say something like `File containing build result from a previous 'skaffold build'`

Reply:
Also not happy with the name but this parameter has existed for a long time in `skaffold deploy`, so I kept the name consistent.

I propose we either keep as-is, or change `skaffold deploy` to support both old and new name roughly like this (untested):

```go
func NewCmdDeploy() *cobra.Command {
	return NewCmd("deploy").
		// [...]
		WithFlags(func(f *pflag.FlagSet) {
			// [...]
			f.VarP(&buildOutputFile, "build-artifacts", "", "Deploy with images in build output file created with `skaffold build --file-output`")
			f.VarP(&buildOutputFile, "build-file", "a", "Deploy with images in build output file created with `skaffold build --file-output`")
			}).
		// [...]
}
```

Reply:
eh, yeah ok I didn't realize this isn't new. I guess let's just leave it the way it's been for now, we can change it later.